framework-reference/navigateto.md

```diff
@@ -2,7 +2,9 @@
 description: >-
   With navigateTo, we can navigate in between the internal pages of Appsmith. It
```

Comment:
navigateTo allows the user to navigate between the internal pages of the App or to an external URL.